### PR TITLE
Prepare 0.7.0 release

### PR DESCRIPTION
**Pull request by snazy**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1427)

### CODE REVIEW

**Comment by snazy** [comment]
@rymurr @nastra anything missing in these two files?

File: site/docs/try/releases.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Releases
 
+## 0.7.0 Release (June 15, 2021)
+
+* Server side filtering for entries-listing
```
Comment (nastra) [inline site/docs/try/releases.md:5]: `Server-side filtering improvements for entries-listing and log-listing`

**Review by nastra — CHANGES_REQUESTED** [review]

File: site/docs/try/releases.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+* Author & committer improvements in the Python CLI
+* Loosen pins for client install on Python cli
+* Gradle plugin improvements
+* (Development) change to Google Code Style, add spotless plugin
+* (CI) Add OWASP Dependency Check
```
Comment (rymurr) [inline site/docs/try/releases.md:10]:
```suggestion
* (CI) Add OWASP Dependency Check
* Fix edge case when merging using in memory store
```

File: python/HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+0.7.0 (2021-06-15)
+------------------
+
+* Author & committer improvements in the Python CLI
+* Fix naming in nessie client merge operation
```
Comment (nastra) [inline python/HISTORY.rst:9]: maybe also mention: `Allow setting author when committing via Python CLI`

File: python/HISTORY.rst [hunk 4]
```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+0.7.0 (2021-06-15)
+------------------
+
+* Author & committer improvements in the Python CLI
```
Comment (nastra) [inline python/HISTORY.rst:8]: maybe `Distinguish between author & committer in the Python CLI` instead?

File: site/docs/try/releases.md (hunk 2 above)
Comment (nastra) [reply site/docs/try/releases.md:10]: should we also mention that we fixed this for the Jgit store?

Comment (snazy) [reply site/docs/try/releases.md:10]: yup - added

**Review by nastra — APPROVED** [review]

**Pull request merged**